src/main/java/io/cdap/plugin/gcp/spanner/sink/SpannerSink.java

```diff
@@ -153,10 +152,8 @@ public void prepareRun(BatchSinkContext context) throws Exception {
     }
   }
 
-  private void createTableIfNotPresent(DatabaseClient dbClient, Database database,
+  private void createTableIfNotPresent(Database database,
                                        Schema schema) throws ExecutionException, InterruptedException {
-    boolean tableExists = isTablePresent(dbClient);
-    if (!tableExists) {
       if (Strings.isNullOrEmpty(config.getKeys())) {
```

Comment:
if condition has been removed, so I think we'll need to fix the indent

Reply:
@rmstar Should we also rename the method to `createTable`? As it no longer checks for existence of table.

Reply:
Yeah sg 👍

Reply:
indent corrected, method renamed